tests/finance/test_slippage.py

```diff
@@ -750,7 +774,8 @@ def test_orders_stop_limit(self):
         bar_data = BarData(self.data_portal,
                            lambda: self.minutes[1],
                            self.sim_params.data_frequency,
-                           self.trading_calendar)
```

Comment:
I've definitely refactored it and added a fixture. Not sure how you're able to comment on a piece of code that's no longer there